tensorflow_datasets/image/icoseg.py

```diff
@@ -0,0 +1,76 @@
+
+"""
+CMU-Cornell iCoseg dataset
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import tensorflow_datasets.public_api as tfds
+
+_CITATION = """
+@inproceedings {
+title = {iCoseg: Interactive Co-segmentation with Intelligent Scribble Guidance},
+author = {D. Batra and A. Kowdle and D. Parikh and J. Luo and T. Chen},
+booktitle = {ECCV},
+year = {2010} }
+"""
+
+_DESCRIPTION = """\
+This is the CMU-Cornell iCoseg dataset for image segmentation. The dataset contains 38 different \
+groups totalling 643 images. These images also have there respective pixel level groundtruth \
+annotations. These groundtruths segment the images into two classes - the subject and the background.
+"""
+
+
+class Icoseg(tfds.core.GeneratorBasedBuilder):
+  """iCoseg dataset for segmentation."""
```

Comment:
Would you consider "for image segmentation" instead of "for segmentation"

Reply:
Sure.